content/basics/use-ably.textile

```diff
@@ -0,0 +1,139 @@
+---
+title: How to use Ably
+meta_description: "A brief introduction on the options available to use with Ably"
+---
+
+Ably provides a number of APIs, SDKs, and adaptors and this page will briefly cover these options available to help you integrate with Ably in the way best suited for you and your business. 
+
+h2(#sdks). Ably SDKs with Realtime and REST Interfaces
+
+All official client library SDKs developed by Ably offer a consistent yet idiomatic API across every language. So regardless of your development environment or language, Ably keeps things simple for you by being consistent and predictable.
+
+The Ably platform also provides a protocol adapters service providing open protocol support and interoperability with a huge array of third party client libraries supporting every imaginable platform.
+
+Ably provides "a number of SDKS":https://ably.com/download in the most popular languages.
+
+h2(#realtime-api). Ably Realtime API
+
+The Ably Realtime client library establishes and maintains a persistent connection to Ably enabling low latency broadcasting and receiving of messages and presence state. All official client library APIs are consistent across every language offering pub/sub functionality, presence, token and basic authentication, active connection management, symmetric encryption and access to channel, presence and metrics history.
+
+Reasons to use the Realtime client library
+
+* You are developing a mobile or web client that needs to subscribe to messages in real time.
+* Your application needs to register its presence on a channel, or listen for others becoming present in real time.
+* You want to maintain a persistent connection to Ably, attach to one or more channels, and publish and subscribe to messages.
+* You want to publish messages at a very high volume, which is much more efficient over a realtime connection, as publishes are pipelined over a websocket.
+* You need to publish messages with very low latency: Realtime publishes have much lower overhead than REST publishes, so have significantly lower latency.
+* You're running into your hourly API request limits. REST publishesr use a separate API request per publish, so if you're hitting your limits there, this may be a reason to use the Realtime library.
```

Comment:
I think this was within a caveat box in the design, but might be worth thinking about in two separate stages - when should you use the SDKs (and when shouldn't you) and then realtime vs REST.